### PR TITLE
Fix object size being 0 when retrying request (#14)

### PR DESCRIPTION
**Pull request by robinst**
Because the code only put the hash in the cache and not the size, the
size that was sent in a retried request was 0.

Some server implementations check the size, so those requests may fail.

To fix it, also put the size in the cache and use it when creating a
request from the cache.

### CODE REVIEW

File: src/main/java/git/lfs/migrate/GitConverter.java [hunk 1]
```diff
@@ -60,7 +60,7 @@ public GitConverter(@NotNull File cachePath, @NotNull File basePath, @NotNull St
         .mmapFileEnable()
         .cacheSoftRefEnable()
         .make();
-    cacheSha256 = cache.getHashMap("sha256");
+    cacheMeta = cache.getHashMap("meta");
```
Comment (robinst) [inline src/main/java/git/lfs/migrate/GitConverter.java]: Changed the name so that it works for old cache files too.

**Comment by bozaro** [comment]
It's really good change. Thanks a lot :+1:

**Comment by bozaro** [comment]
Sorry for delay. I was offline for two weeks.

**Comment by robinst** [comment]
No worries, thanks! :)

**Pull request merged**